src/transforms/rename.jl

```diff
@@ -6,6 +6,17 @@
     Rename(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ)
 
 The transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+
+# Examples
+
+```julia
+T = Rename(Dict(:a => :x, :c => :y))
```

Comment:
@juliohm, I think the `Rename(dict)` method is part of the internal API, should we expose this to the user?

Reply:
Good catch @eliascarv , I think we should omit it. Advanced users can always read the source. Agree?